server/src/Core.hs

```diff
@@ -76,21 +74,20 @@ data Core = Core
   , coreMetrics      :: Maybe Metrics.IcepeakMetrics
   }
 
-data SubscriberState = SubscriberState
-  { subscriberConnection :: WS.Connection
-  , subscriberQueue      :: TBQueue Value
-  }
+-- This structure describes the state associated to each subscriber in order to
+-- communicate with them, at the moment, a simple bounded queue. This is a
```

Comment:
I would explain why we need the bounded queue here (=> because every subscriber can respond at a a different speed, and we don't want to wait until the slowest subscriber has responded. And the queue is bounded because we don't want to have too many messages pile up, since that would be a DOS vector)